onboarding/asset-management-directions.md

```diff
@@ -0,0 +1,126 @@
+
+To have the new bibliography be included on the website, you can do one of two things. You can manually export the bib file and drop it and save it as```\_bibliography\references.bib,``` or alternatively, you can use an add-on such as [better BibTex](https://retorque.re/zotero-better-bibtex/) that automatically exports your bib file. Once you are ready to push your new bib file onto the website,  follow [these steps](#github-and-jekyll-workflow) on how to do so. We use [Jekyll Scholar](https://github.com/inukshuk/jekyll-scholar) to manage the reference on the webfront (hence why the standardized tags!). It allows you to include subset of literature on the website based on fields in the bibfile.
+
+### Reference tags
+
+| Tag       | Include if         |
+| ------------- |:-------------:|
+| Monocyte derived dendritic cell| Related to cell type |
+| Dendritic cell| Related to cell type    |  
+| Monocyte derived macrophage| Related to cell type      |  
+| Alveolar macrophage| Related to cell type     |
+| Alveolar epithelial cell| Related to cell type      |  
+| Airway epithelial cell| Related to cell type     |
+| Aspergillus fumigatus| Related to the fungus itself    |
+| Neutrophil| Related to cell type     |  
+| Innate immunity| Overview of innate immunity response |
+| Software| Related to software used in the project |  
+| Review paper| Reference is a review paper|
+| Invasive Pulmonary Aspergillosis| Paper specific to IPA    |
+| In vitro| Includes _in vitro_ data used to calibrate a model|
+| In vivo| Includes _in vivo_ data used to calibrate a model|
+| Multiscale modeling| Reference related to techniques of multiscale modeling|
+| Preprocessing tools| Related to a tool used in the preprocessing step     |
+| Bioinformatics tool|  Related to a tool used in the bionformatics analysis    |
+| Our publication| Papers that cite funding related to these projects     |
+| Intracellular model| Paper is used in the building of an intracellular network model in this project |
+| Parameter reference| A parameter related to a model is derived from this paper  |
+| Human| Includes data from humans  |
+| Mouse| Includes data from mice |
+| Numerical methods| Paper related to numerical method     |
+| Software architecture| Paper related to software architecture    |
+| Information management | Paper related to information management    |
+| Transcriptomic dataset | Paper includes a transcriptomic dataset|
+
+## Our suggested Github workflow
+
+
+### Github and Jekyll workflow
+
+## Adding assets onto Girder
```

Comment:
i assume this will mention tags? we should also have a section on search. though we know both of these will change soon.